### PR TITLE
Update page template descriptions

### PR DESCRIPTION
**Pull request by anandamaryon1**
## Checklist

- [ ] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/main/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/main/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ ] CHANGELOG entry
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry

### CODE REVIEW

**Comment by frankieroberto** [comment]
@anandamaryon1 this has lost the updates to the page template itself somehow?

**Comment by anandamaryon1** [comment]
> @anandamaryon1 this has lost the updates to the page template itself somehow?

Argh, maybe I messed up a merge? Or maybe it's not in this PR because the changes are already on the branch? (i.e. I made exactly the same changes separately?)

**Comment by anandamaryon1** [comment]
Ah good, was worried I'd messed something up. 

I incorporated your wording in this PR 👍  (sorry it's all a bit hard to follow, need to work on our processes). Merging now and I'll double check it all.

**Comment by frankieroberto** [comment]
@anandamaryon1 auto-deploy from `main` instead of doing release branches? 😄
